tests/test_io.py

```diff
@@ -66,12 +66,14 @@ def test_upload_artifacts_to_azure_wasb(dummy_kwargs):
         assert hook_instance.load_file.call_count == 2
 
 
+@pytest.mark.integration
```

Comment:
same question as above. do we know what part of this test needs an integration? i am hoping if we could mock something more that maybe missing to be mocked and keep this as a unit test? Maybe we could add a TODO for this to recheck later.

Reply:
With the new mocks, this became a unit test 🎉